### PR TITLE
CU-86drtv2jj - NEON3 - Portfolio - Design Enhancements (New side bar) and bar chart improvements

### PR DESCRIPTION
**Pull request by Thacryba**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by melanke** [comment]
Task linked: [CU-86drtv2jj NEON3 - Portfolio - Design Enhancements (New side bar) and bar chart improvements](https://app.clickup.com/t/86drtv2jj)

File: src/renderer/src/routes/pages/Portfolio/Activity/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { TransactionsTable } from '@renderer/components/BlockchainDataPanel/TransactionsTable'
+import { Separator } from '@renderer/components/Separator'
+import { BalanceHelper } from '@renderer/helpers/BalanceHelper'
+import { FilterHelper } from '@renderer/helpers/FilterHelper'
+import { useAccountsSelector } from '@renderer/hooks/useAccountSelector'
+import { useBalancesAndExchange } from '@renderer/hooks/useBalancesAndExchange'
+import { useWalletsSelector } from '@renderer/hooks/useWalletSelector'
+
+export const PortfolioActivityPage = () => {
+  const { t } = useTranslation('components', { keyPrefix: 'portfolioActivity' })
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Portfolio/Activity/index.tsx]: change this translation from components to pages namespace

File: src/renderer/src/routes/pages/Portfolio/Overview/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { BalanceChart } from '@renderer/components/BalanceChartPanel/BalanceChart'
+import { Separator } from '@renderer/components/Separator'
+import { BalanceHelper } from '@renderer/helpers/BalanceHelper'
+import { FilterHelper } from '@renderer/helpers/FilterHelper'
+import { useAccountsSelector } from '@renderer/hooks/useAccountSelector'
+import { useBalancesAndExchange } from '@renderer/hooks/useBalancesAndExchange'
+import { useWalletsSelector } from '@renderer/hooks/useWalletSelector'
+
+export const PortfolioOverviewPage = () => {
+  const { t } = useTranslation('components', { keyPrefix: 'portfolioOverview' })
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Portfolio/Overview/index.tsx]: change this translation from components to pages namespace

File: src/renderer/src/routes/pages/Portfolio/index.tsx [hunk 3]
```diff
@@ -1,27 +1,51 @@
...
+          <Separator />
+        </div>
+        <ul className="max-w-full w-full">
+          <PortfolioSidebarLink title={t('overview')} to="/portfolio/overview" />
+          <PortfolioSidebarLink title={t('allActivity')} to="/portfolio/activity" />
+          <PortfolioSidebarLink
+            title={t('allConnections')}
+            type="button"
+            onClick={modalNavigateWrapper('dapp-connection-list')}
+          />
```
Comment (raulduartep) [inline src/renderer/src/routes/pages/Portfolio/index.tsx]: You should verify if WalletConnect is initialized to let the user clicks here.
```suggestion
import { EStatus } from '@cityofzion/wallet-connect-sdk-wallet-core'
import { useWalletConnectWallet } from '@cityofzion/wallet-connect-sdk-wallet-react'

  const { status } = useWalletConnectWallet()
  
          <PortfolioSidebarLink
            title={t('allConnections')}
            type="button"
            disabled={status !== EStatus.STARTED}
            onClick={modalNavigateWrapper('dapp-connection-list')}
          />
```

**Pull request merged**